docs/AUTHORS.md

```diff
@@ -16,5 +16,6 @@
 * Oflor
 * Robbie Cooper
 * Scott Corbeil
+* Thomas Schaller
```

Comment:
Might as well add everyone that got a PR merged while we are at it :)

-Joël Lupien
-Hittherhod
-Matthias Schuster
-Simon Rönnberg